src/utilities/vulnerabilityAndSignatureCheck.js

```diff
@@ -0,0 +1,89 @@
+import React from 'react';
+import {
+  NoneVulnerabilityIcon,
+  UnknownVulnerabilityIcon,
+  LowVulnerabilityIcon,
+  MediumVulnerabilityIcon,
+  HighVulnerabilityIcon,
+  CriticalVulnerabilityIcon,
+  NoneVulnerabilityChip,
+  UnknownVulnerabilityChip,
+  LowVulnerabilityChip,
+  MediumVulnerabilityChip,
+  HighVulnerabilityChip,
+  CriticalVulnerabilityChip,
+  UnverifiedSignatureIcon,
+  VerifiedSignatureIcon,
+  UnverifiedSignatureChip,
+  VerifiedSignatureChip
+} from './vulnerabilityAndSignatureComponents';
+
+const vulnerabilityIconCheck = (vulnerabilitySeverity) => {
```

Comment:
Based on the fact that these functions are wrappers over the components and they are functions that return JSx, i think it would be better for `vulnerabilityIconCheck` `vulnerabilityChipCheck` `signatureIconCheck`  and `signatureChipCheck` to be React components themselved, and have the current params passed as props